genai-on-vertex-ai/genai-website-mod/backend/main.py

```diff
@@ -78,6 +79,29 @@ def get_state_by_action(action: str) -> ContentState:
     else:
         return ContentState.PRODUCTION
 
+def execute_search_ask_video(thequery: str):
+    """
+    This is a "Ask Video" feature of execute_first_search. As one of two
+    implementation of the web search feature in this app.
+    """
+    request_data = VertexLLMVideoChatRequest(
+        prompt=thequery,
+        video_url="https://www.youtube.com/watch?v=St1up9PwnUY",
```

Comment:
Is this video url supposed to be hardcoded ?

Reply:
It's intentional.
Move the url to `config.toml` with a default value points to "https://www.youtube.com/watch?v=St1up9PwnUY".